sdks/java/io/kinesis/src/test/java/org/apache/beam/sdk/io/kinesis/ShardReadersPoolTest.java

```diff
@@ -41,11 +41,11 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.mockito.stubbing.Answer;
 
 /** Tests {@link ShardReadersPool}. */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
What does this change?

Reply:
This test has some unfinished stubbings, which I silenced for now (equivalent to Mockito 1 behavior). A code maintainer should take a look and evaluate if the stubbings are actually unused and resolve it accordingly.

Reply:
OK, cool.